meshnets/data_processing/data_mappers.py

```diff
@@ -54,3 +54,19 @@ def make_edge_features(example):
     example['edge_features'] = np.concatenate(
         [displacement_vectors, distances[:, None]], axis=1)
     return example
+
+
+def make_node_features(example):
+    """Makes the node features from the node coordinates.
+
+    The node features:
+
+    - The wind vector. This is a common value for all the nodes in the
+      graph.
+
+    """
+    nodes = np.array(example['nodes'])
```

Comment:
why do you need to convert this to numpy? Seems like the only thing you need is the number of nodes to iterate on...  Can you do num_nodes = len(example['nodes']) or similar?